src/library_wasmfs_fetch.js

```diff
@@ -12,9 +12,12 @@ mergeInto(LibraryManager.library, {
   _wasmfs_create_fetch_backend_js__deps: [
     '$wasmFS$backends',
     '$wasmFS$JSMemoryFiles',
+    '$wasmFS$backendFileNames',
     '_wasmfs_create_js_file_backend_js',
   ],
-  _wasmfs_create_fetch_backend_js: async function(backend) {
+  _wasmfs_create_fetch_backend_js: async function(backend, baseUrl) {
+    var _baseUrl = UTF8ToString(baseUrl);
+    var isAbs = _baseUrl.indexOf('://') !== -1;
```

Comment:
This is the naming scheme I've used for similar parameters in other WasmFS backends.
```suggestion
  _wasmfs_create_fetch_backend_js: async function(backend, baseUrl_p) {
    var baseUrl = UTF8ToString(baseUrl_p);
    var isAbs = baseUrl.indexOf('://') !== -1;
```

Reply:
Done.